### PR TITLE
DistributeLockContext.renew(final AckLockedEvent event) need add unit test

### LINKED ISSUE #16647: DistributeLockContext.renew(final AckLockedEvent event) need add unit test

### PR DESCRIPTION
**Pull request by weihubeats**
Fixes #16647.

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/lock/DistributeLockContextTest.java [hunk 1]
```diff
@@ -42,7 +43,7 @@
 import static org.mockito.Mockito.mock;
 
 public final class DistributeLockContextTest {
-    
+
```
Comment (yx9o) [inline shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/lock/DistributeLockContextTest.java:46]: Please keep the indentation.

Comment (weihubeats) [reply shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/lock/DistributeLockContextTest.java]: ok processed

Comment (yx9o) [reply shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/lock/DistributeLockContextTest.java]: Sorry, there are still some unfinished, please check the file again.
image

Comment (weihubeats) [reply shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/lock/DistributeLockContextTest.java]: ok, you're looking, thanks

**Review by yx9o — APPROVED** [review]

**Pull request merged**